src/sass/utils/_variables.scss

```diff
@@ -2,12 +2,17 @@
 =============================================*/
```

Comment:
ech ... trochę słabo zmieniać kolory w połowie projektu ale widze skoro jest to push na main to trudno trzeba zaktualizować